lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -46,6 +50,7 @@
                        _.each([
                            [':default', _.bind(this.routeNotFound, this)],
                            ['topics/:topic_id', _.bind(this.browseTopic, this)],
+                           ['topics/:topic_id/create_new_team(/)', _.bind(this.newTeam, this)],
```

Comment:
I think that this route should be renamed to `'topics/:topic_id/create-team(/)'`. Hyphens are preferred to underscores (best practice), and "new" is redundant.